milatools/datasets/torch.py

```diff
@@ -0,0 +1,23 @@
+import os
+import subprocess
+
+
+def fetch_imagenet(local_directory=None):
+    if "SLURM_JOB_ID" in os.environ.keys():
```

Comment:
I think this may be too brittle, because `SLURM_JOB_ID` is going to exist on all clusters that use SLURM. I think this is probably the best way:

```python
import socket
if socket.getfqdn().endswith(".server.mila.quebec") and "SLURM_TMPDIR" in os.environ.keys():
    ...
```